tool/jt.rb

```diff
@@ -2034,8 +2041,9 @@ def graph(*args)
     raise unless test_file
     env = ruby_running_jt_env
 
-    if Gem::Specification.find_all_by_name('seafoam').empty?
-      sh env, 'gem', 'install', 'seafoam'
+    seafoam_version = '0.11'
+    unless Gem::Specification.find_all_by_name('seafoam').any? { |g| g.version == Gem::Version.new(seafoam_version) }
+      sh env, 'gem', 'install', 'seafoam', '-v', seafoam_version
```

Comment:
This won't work in the internal CI (no internet access, should not install gems on the ruby in PATH).
Instead it should be done like `rubocop` with a `seafoam` helper method.
And then I'll add it to gem_test_pack.

Reply:
I can do this part